### PR TITLE
[bitnami/mlflow] feat: 🎉 Add chart

### PR DESCRIPTION
**Pull request by javsalgar**
### Possible drawbacks

This PR adds a helm chart for MLflow. Has the following features:

- TLS support
- Network policies
- Prometheus metrics
- HPA and VPA
- Bitnami standards

### Applicable issues

- fixes #

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/mlflow/templates/run/source-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{{- /*
+Copyright VMware, Inc.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if (include "mlflow.v0.run.source.createConfigMap" .) }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ printf "%s-source" (include "common.names.fullname" .) | trunc 63 | trimSuffix "-" }}
```
Comment (fmulero) [inline bitnami/mlflow/templates/run/source-configmap.yaml:10]: Shouldn't it be?
```suggestion
  name: {{ include "mlflow.v0.run.source.configMapName" . }}
```

**Review by fmulero — CHANGES_REQUESTED** [review]
Great @javsalgar!

I left some comments, could you give them a glance?

File: .vib/mlflow/goss/goss.yaml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    filetype: directory
+    mode: '2775'
+    owner: root
+  /bitnami/mlflow-basic-auth/basic_auth.ini:
+    exists: true
+    filetype: file
+    mode: '0644'
+    contains:
+      - /admin_username={{ .Vars.tracking.auth.username }}/
+      - /admin_password={{ .Vars.tracking.auth.password }}/
```
[the remark this answers is not in the document]
Comment (javsalgar) [reply .vib/mlflow/goss/goss.yaml:16]: This is necessary for random users (a.k.a openshift) to work

**Review by fmulero — APPROVED** [review]
Great @javsalgar LGTM

**Pull request merged**